pydyf/__init__.py

```diff
@@ -364,6 +365,42 @@ def transform(self, a, b, c, d, e, f):
             _to_bytes(a), _to_bytes(b), _to_bytes(c),
             _to_bytes(d), _to_bytes(e), _to_bytes(f), b'cm')))
 
+    def inline_image(self, width, height, bpc, raw_data):
+        """Add an inline image.
+
+        :param width: The width of the image.
+        :type width: :obj:`int`
+        :param height: The height of the image.
+        :type width: :obj:`int`
+        :param bpc: The bits per component. 1 for BW, 8 for grayscale.
+        :type width: :obj:`int`
+        :param raw_data: The raw pixel data.
+
+        """
+        if self.compress:
+            data = zlib.compress(raw_data)
+        else:
+            data = raw_data
+        enc_data = base64.a85encode(data).decode("ascii")
+        self.stream.append(
+            " ".join(
```

Comment:
The join should be on `b' '` instead of `" "` because, in the end, we’ll have bytes written.